sections/list-collections.liquid

```diff
@@ -45,39 +57,98 @@
   {%- endcase -%}
 
   {%- case padding_bottom_mobile -%}
-    {%- when 'none' -%}
-      --padding-bottom-mobile: 0;
     {%- when 'small' -%}
       --padding-bottom-mobile: 18px;
     {%- when 'medium' -%}
       --padding-bottom-mobile: 40px;
     {%- when 'large' -%}
       --padding-bottom-mobile: 60px;
   {%- endcase -%}
+
+  {%- case grid -%}
+    {%- when 'two' -%}
+      --template-columns: 2;
+    {%- when 'three' -%}
+      --template-columns: 3;
+  {%- endcase -%}
+
+  {%- case spacing -%}
+    {%- when 'none' -%}
+      --grid-gap: 0;
+    {%- when 'small' -%}
+      --grid-gap: 4px;
+    {%- when 'medium' -%}
+      --grid-gap: 16px;
+    {%- when 'large' -%}
+      --grid-gap: 32px;
+  {%- endcase -%}
+
+  {%- case spacing_mobile -%}
+    {%- when 'none' -%}
+      --grid-gap-mobile: 0;
+    {%- when 'small' -%}
+      --grid-gap-mobile: 4px;
+    {%- when 'medium' -%}
+      --grid-gap-mobile: 16px;
+    {%- when 'large' -%}
+      --grid-gap-mobile: 32px;
+  {%- endcase -%}
+
+  {%- case rounded_corners -%}
+    {%- when 'none' -%}
+      --border-radius-rounded-blocks: 0px;
+    {%- when 'small' -%}
+      --border-radius-rounded-blocks: 8px;
+    {%- when 'medium' -%}
+      --border-radius-rounded-blocks: 16px;
+    {%- when 'large' -%}
+      --border-radius-rounded-blocks: 32px;
+  {%- endcase -%}
+
+  {%- if show_overlay -%}
+    --opacity-overlay: {{ overlay_opacity | append: "%"}};
```

Comment:
Is there a reason for not adding this to the condition on line 26?

Reply:
There is no reason, I just missed it
Combined into 1 place